react-exercise/src/usersClientService.ts

```diff
@@ -0,0 +1,50 @@
+import { fetchUsers, createUser, updateUser as updateUserOnServer, deleteUser } from './usersDBClient';
+import { addUser, setUsers, updateUser as updateUserOnStore, removeUser } from './stores/usersStore';
+import { User } from './types/user';
+
+// Sync all users from server to Zustand store
+export const fetchUsersSync = async (
```

Comment:
Why "Sync" in the name ? 
It's a bit misleading an makes the reader think this method is synchronic

Reply:
I thought that "fetch" maybe not convey good enough the fact that i'm syncing the data with the users store as well, but I guest this is part of the abstraction of the user anyway so I can remove it?